distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFederation.java

```diff
@@ -2500,6 +2506,78 @@ private void checkExpectations(CometDClient cometDClient, Integer numNotificatio
 
     }
 
+    /**
+     * Confirms that the "download resource to local cache" action's url can be
+     * found in the CometDClient messages and matches the expected url.
+     *
+     * @throws Exception
+     */
+    @Test
+    public void findMetacardActionUrlUsingCometDClient() throws Exception {
```

Comment:
We can probably come up with a better method name for this.  Maybe retrieveAsyncDownloadActionUrlFromSearchEndpoint.